v2-components/Icons/ChevronUp/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { Icon, IconProps } from '@chakra-ui/react';
+
+interface ChevronUpProps extends IconProps {
+  width?: number | string;
+  height?: number | string;
+}
+
+const ChevronUp = ({ width = '20px', height = '20px', ...props }: ChevronUpProps) => {
+  return (
+    <Icon width={width} height={height} color="cyan.400" viewBox="0 0 25 24" fill="none" {...props}>
+      <path
```

Comment:
I think it would make sense to rely on Chakra icons for v2 stuff?
https://chakra-ui.com/docs/components/icon

Reply:
As far as I know we are using [font awesome](https://www.figma.com/file/uFHKfLFYczHnZ7voSluRvO/Chakra-UI-SNX-System?node-id=441%3A1738)

Reply:
Lol. Chakra does have icons (including chevron), but not that many. We'd be reaching often for Icons from a different set. https://chakra-ui.com/docs/components/icon/usage#all-icons

Reply:
Yea ok I just saw that v3 added chakra icons too. Wanna open an discussion with designers and v3 so we align on icons?